### PR TITLE
Permalinks docs

### PR DESCRIPTION
**Pull request by GenaBitu**
I've replaced the README section about deep links with history
I've moved the demo down and split it into a history demo and a deep links demo

Also, the deep links demo doesn't work for me - I think it is a bug with the history code

### CODE REVIEW

**Comment by GenaBitu** [comment]
Hey, and thanks for the WP plugin plug! I've only now noticed

**Comment by GenaBitu** [comment]
I've fixed the bug

File: docs/javascripts/main.js [hunk 1]
```diff
@@ -77,16 +73,13 @@ $(document).ready(function() {
...
-        var lightboxInstance = $('a[data-imagelightbox="k"]').imageLightbox();
-        lightboxInstance.startImageLightbox(image);
-    }
+    $('a[data-imagelightbox="j"]').imageLightbox({
+        history: true
+    });
+
+    $('a[data-imagelightbox="k"]').imageLightbox({
+        history: true
+    });
```
Comment (rejas) [inline docs/javascripts/main.js]: twice the exmaple, twice the fun? ;-)

Comment (GenaBitu) [reply docs/javascripts/main.js]: Well, once it is with `data-ilb2-id` and once without it...

Comment (GenaBitu) [reply docs/javascripts/main.js]: I've added a better explanation in 55b91e8

**Pull request merged**